src/SFA.DAS.EmployerAccounts/Services/AuditApiClientForSupportUser.cs

```diff
@@ -0,0 +1,43 @@
+﻿using SFA.DAS.Audit.Client;
+using SFA.DAS.Audit.Types;
+using SFA.DAS.Authentication;
+using System.Threading.Tasks;
+using SFA.DAS.EmployerAccounts.Extensions;
+using SFA.DAS.EmployerUsers.WebClientComponents;
+using System.IdentityModel.Claims;
+using System.Collections.Generic;
+
+namespace SFA.DAS.EmployerAccounts.Services
+{
+    public class AuditApiClientForSupportUser : IAuditApiClient
+    {
+        private readonly IAuditApiClient _client;
+        private readonly IAuthenticationService _authenticationService;
+
+        public AuditApiClientForSupportUser(IAuditApiClient client, IAuthenticationService authenticationService)
+        {
+            _client = client;
+            _authenticationService = authenticationService;
+        }
+        public Task Audit(AuditMessage message)
+        {
+            if (_authenticationService.IsSupportUser())
+            {
+                var impersonatedUser = _authenticationService.GetClaimValue(DasClaimTypes.Id);
```

Comment:
I don't think that these two claims are being populated with the impersonated users information as they would normally be when coming through the staff support user route.

Reply:
These were the claims that were setup on the identity when Vas logged on as a staff user.

Reply:
ok I think we need to look at this when testing alongside the other branches because I'm not sure these claims should be being set when logging in via staff idams

Reply:
approving as is, but may need reviewing during testing